contents/blog/open-source-eating-saas.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2020-08-17
+title: Open source is eating SaaS
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+Open source has several key advantages of traditional Software-As-A-Service (SaaS) business models. We think most companies building software will launch this way in the future.
+
+## It's far easier to build something useful
+
+### More people will try it out
+
+In the early days, startups are *ravenous* for feedback. It's the easiest way to build something that people want. It's very, very unlikely you'll hit the nail on the head with the first thing that you launch - it'll invariably need iteration or a complete pivot.
+
+You know those [generic looking launches](https://www.google.com/search?q=startup+one+page+template&rlz=1C5CHFA_enGB833GB833&source=lnms&tbm=isch&sa=X&ved=2ahUKEwiT8a_yk6LrAhVZShUIHYpwAMcQ_AUoAXoECAwQAw&biw=1745&bih=1832)? E8ven if the product looks awesome, the thought of security issues with how your data is handled, or the service disappearing in a few months scare off many technical users.
+
+If your software is hosted only, then users have to be willing to give up some of their time *and* somewhere from a little to a lot of their data in exchange for feedback, as a minimum. The less stuff you have to ask for, the easier it is to get a yes. Open source means your users can self host if they want to, so now it's just a time thing.
+
+More usage means more feedback, especially when places like GitHub are *designed* for people to raise issues / feature requests. The vast majority of open source users already have a GitHub account and can just raise an issue. SaaS product feedback in the early days usually involves emails/phone calls/Slack groups. Open source has all of this available *and* their community of users are already used to leaving feedback in the repo.
+
+Just look at the YCombinator launches for open source to get a sense of what gets a good reception from the HackerNews community:
+
+*INSERT SOME DATA BASED ON LAUNCH HNS*
+
+### Your work is public
+
+Goodbye hardcoding stuff. It's easier to take pride in your work when it's completely visible. This can make shipping a little slower, but we've found this is outweighed by new developers finding the codebase much easier to get started with - let alone the value in proper documentation.
+
+## Building a better team
+
+Open source attracts more developers.
+
+Open source carries a sense of purpose automatically - it's pretty nifty being able to say anyone in the world is welcome to use or modify what you're working on, even if they've no money to spend. It's gratifying to be offering something for free that lots of people can find useful.
```

Comment:
I personally prefer to write out "have" in most scenarios, unless it's paired with I (I've). i.e. I would prefer "they have", because "they've" gives me a feeling that I should add "got" after it.